dex/models/trades/arbitrum/platforms/maverick_v2_arbitrum_base_trades.sql

```diff
@@ -0,0 +1,21 @@
+{{
+    config(
+        schema = 'maverick_v2_arbitrum',
```

Comment:
We may need to add the model schema for these new spells.

Reply:
Something like this:
https://github.com/duneanalytics/spellbook/blob/main/dex/models/trades/arbitrum/_schema.yml#L722

Reply:
@Hosuke  Great; I made the changes.

I haven't been able to debug the build failure.  

<img width="1239" alt="image" src="https://github.com/user-attachments/assets/246de62c-848b-4f41-9e15-e1acd8fec1dc">

Seems that there is some issue with the seed. But all seems fine in the file.  Any ideas?

Reply:
We need to add the schema for seed files as well.